runtime/nls/shrc/j9shr_ca.nls

```diff
@@ -6913,3 +6908,36 @@ J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.explanation=Th
 J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.system_action=The JVM continues.
 J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.user_response=Contact your service representative.
 # END NON-TRANSLATABLE
+
+J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1=La mida de p\u00e0gina ( %1$zu ) del sistema operatiu \u00e9s diferent de la mida de p\u00e0gina ( %2$zu ) del sistema en qu\u00e8 es crea la mem\u00f2ria cau de la capa %3$d .
```

Comment:
Did the automation add the spaces inside the parentheses? I don't think we want them. They're not added universally, e.g. they're not added for `de`, `fr` or `it`.

Reply:
Yes the automation added them. We could remove them manually but then would always have to fix this up when doing future translations, which I don't want to do. I suggest we just accept them, they aren't harmful.

Reply:
This is disappointing. There are other places in this same file (e.g. line 6293) where those spaces were not added, so we can't say it's a custom of certain languages. I think you're right that they aren't harmful, but I don't know for sure. Can you follow-up with the maintainers of the translation tool to see whether inconsistencies like this are addressed?

Reply:
Good news, I discovered how to manually update translations, see https://github.com/eclipse-openj9/openj9/pull/20230